chevah/compat/__init__.py

```diff
@@ -30,6 +30,9 @@
     DefaultAvatar = UnixDefaultAvatar
     SuperAvatar = UnixSuperAvatar
 
+    # Unconditionally allow cryptography 3.2.1 with OpenSSL 1.0.2.
+    os.environ['CRYPTOGRAPHY_ALLOW_OPENSSL_102']  = 'yes'
```

Comment:
I guess that lint will complain about this:

```suggestion
    os.environ['CRYPTOGRAPHY_ALLOW_OPENSSL_102'] = 'yes'
```

Reply:
Yes, thanks, I have already fixed that locally.